src/shared/reducers/page/submission/index.js

```diff
@@ -39,10 +39,10 @@ function onSubmitDone(state, { error, payload }) {
   /* TODO: I am not sure, whether this code is just wrong, or does it handle
    * only specific errors, returned from API for design submissions? I am
    * adding a more generic failure handling code just above. */
-  if (payload.error) {
+  if (payload.result && !payload.result.success) {
```

Comment:
@sumitdaga I don't think this will do it. The v5 payload does not put errors in the response like that. We need to handle the issue on the HTTP Status codes